### PR TITLE
Better setup documentation and errors

### PR DESCRIPTION
**Pull request by AutomatedTester**
I have added documentation on how to run tests and updated error messages to be more meaningful.

### CODE REVIEW

**Comment by AutomatedTester** [comment]
We need to decide on #120 which should really be py2 though I have written it should be py3 in this PR

File: run/run.py [hunk 1]
```diff
@@ -361,7 +361,7 @@ def get_config():
     ]
     # Expand paths, this is for convenience so you can use $HOME
     for key in expand_keys:
-        config.set('default', key, os.path.expandvars(config['default'][key]))
+        config.set('default', key, os.path.expandvars(config.get('default',key)))
```
Comment (jeffcarp) [inline run/run.py]: Can you add a space after `,`?

File: README.md [hunk 2]
```diff
@@ -20,6 +20,22 @@ curl http://localhost:8080/tasks/populate-dev-data
...
+
+You'll need to make sure that you have Python 3 installed. It is recommended that you setup a [virtualenv](https://virtualenv.pypa.io/en/stable/). When you have activated your `virtualenv`
+run the following command
+```sh
+  pip install -r requirements
+```
+
+### Running
+
+Copy the file `run/running.example.ini` and edit the fields to the correct locations for items on your machine.
```
Comment (jeffcarp) [inline README.md]: I think this should specify the destination file, e.g. copying from `run/running.example.ini` to `run/running.ini`.

File: README.md [hunk 3]
```diff
@@ -20,6 +20,22 @@ curl http://localhost:8080/tasks/populate-dev-data
...
+If you do not do this you will receive a `configparser.NoSectionError: No section: 'default'` message in your
+console.
```
Comment (jeffcarp) [inline README.md]: I think instead of having this sentence, there should be a warning in `run/run.py` if the file is not detected.

**Comment by AutomatedTester** [comment]
comments addressed.

**Comment by jeffcarp** [comment]
👍 much appreciated!

**Pull request merged**